eks-anywhere-common/Testers/dynatrace/testJob.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: dynatrace-tester-cron
+  namespace: dynatrace
+spec:
+  schedule: "*/10 * * * *"
```

Comment:
Fix the Cron Schedule pls something like this `10 10 * * *`.

Reply:
adjusted the frequency as suggested!